### PR TITLE
chore: Rework NFTs

### PR DESCRIPTION
**Pull request by RabbitDoge**
[ ] Before opening a pull request, please read the [contributing guidelines](https://github.com/pancakeswap/pancake-frontend/blob/master/CONTRIBUTING.md) first
[ ] If your PR is work in progress, open it as `draft`
[ ] Before requesting a review, all the checks need to pass
[ ] Explain what your PR does

### CODE REVIEW

File: src/config/constants/nfts/types.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  description: string
+  images: NftImages
+  video?: {
+    webm: string
+    mp4: string
+  }
+
+  // Uniquely identifies the nft.
+  // Used for matching an NFT from the config with the data from the NFT's tokenURI
+  identifier: string
```
Comment (hachiojidev) [inline src/config/constants/nfts/types.ts:32]: We  can probably remove this and use the `id` now.

File: src/state/collectibles/hooks.ts [hunk 2]
```diff
@@ -23,6 +24,6 @@ export const useGetCollectibles = () => {
     isInitialized,
     isLoading,
     tokenIds: data,
-    nftsInWallet: Nfts.filter((nft) => identifiers.includes(nft.identifier)),
+    nftsInWallet: Nfts.pancake.filter((nft) => identifiers.includes(nft.identifier)),
```
Comment (hachiojidev) [inline src/state/collectibles/hooks.ts:27]: This gets it from the pancake collection. Perhaps we should rename this hook to `useGetPancakeCollection` or something.

File: src/state/collectibles/index.ts [hunk 3]
```diff
@@ -20,8 +20,8 @@ export const fetchWalletNfts = createAsyncThunk<NftSourceItem[], string>(
   'collectibles/fetchWalletNfts',
   async (account) => {
     // For each nft source get nft data
-    const nftSourcePromises = Object.keys(nftSources).map(async (nftSourceType) => {
-      const { address: addressObj } = nftSources[nftSourceType as NftType]
+    const nftSourcePromises = Object.keys(collections).map(async (nftSourceType) => {
+      const { address: addressObj } = collections[nftSourceType as CollectionKey]
```
Comment (hachiojidev) [inline src/state/collectibles/index.ts:24]: Instead of typing it here, maybe type it on line 23.

File: src/utils/collectibles.ts [hunk 4]
```diff
@@ -27,8 +16,8 @@ export const getTokenUrl = (tokenUri: string) => {
   return tokenUri
 }
 
-export const getAddressByType = (type: NftType) => {
-  return getAddress(nftSources[type].address)
+export const getBunnyNftAddress = () => {
+  return getAddress(collections.pancake.address)
```
Comment (hachiojidev) [inline src/utils/collectibles.ts:20]: Same here - maybe we should pass in a collection or something.

File: src/config/constants/nfts/types.ts (hunk 1 above)
Comment (hachiojidev) [reply src/config/constants/nfts/types.ts:32]: Not a priority of course.

Comment (RabbitDoge) [reply src/config/constants/nfts/types.ts:32]: It will change the state (a big rework), and this code will be removed on the nft market anyway

**Review by hachiojidev — APPROVED** [review]

**Comment by willianr2304** [comment]
Get

**Comment by willianr2304** [comment]
Deposit

**Pull request merged**